packages/sit-onyx/src/components/OnyxPageLayout/OnyxPageLayout.stories.ts

```diff
@@ -0,0 +1,144 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxPageLayout from "./OnyxPageLayout.vue";
+
+/**
+ * Layout component that structures your page.
+ * Includes space for a sidebar, page content, footer and toasts.
+ * Recommended to use on view level of an application.
+ */
+const meta: Meta<typeof OnyxPageLayout> = {
+  title: "layout component/OnyxPageLayout",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxPageLayout,
+    events: [],
+    argTypes: {
+      default: {
+        control: { disabled: true },
+      },
+      sidebar: {
+        control: { disabled: true },
+      },
+      footer: {
+        control: { disabled: true },
+      },
+      toasts: {
+        control: { disabled: true },
+      },
+    },
+  }),
+  // storybook adds 1rem padding. The app layout fills the full available space
+  // so we need to counteract the padding with a negative margin.
+  decorators: [
+    (story) => ({
+      components: { story },
+      template: `
+        <div style="margin: -1rem; height: 20rem;" >
+          <story />
+        </div>`,
+    }),
+  ],
+  render: (args) => ({
+    setup: () => ({ args }),
+    ...getPageRenderContent(),
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxPageLayout>;
+
+/** A standard page with some content. */
+export const Default = {
+  args: {},
+} satisfies Story;
+
+/** A standard page with a fixed sidebar. */
+export const WithSidebar = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => ({
+    setup: () => ({ args }),
+    ...getPageRenderContent({ sidebar: true }),
+  }),
+} satisfies Story;
+
+/** A standard page with a footer. */
+export const WithFooter = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => ({
+    setup: () => ({ args }),
+    ...getPageRenderContent({ footer: true }),
+  }),
+} satisfies Story;
+
+/** A page that shows a sidebar and a footer next to it. */
+export const WithPartialFooter = {
+  args: {
+    ...Default.args,
+    footerAsideSidebar: true,
+  },
+  render: (args) => ({
+    setup: () => ({ args }),
+    ...getPageRenderContent({ sidebar: true, footer: true }),
+  }),
+} satisfies Story;
+
+/** A page that shows a toast. */
+export const WithToast = {
+  args: {
+    ...Default.args,
+    footerAsideSidebar: true,
+  },
+  render: (args) => ({
+    setup: () => ({ args }),
+    ...getPageRenderContent(
+      { sidebar: true, footer: true },
+      `
+    <template #toasts>
+      <div style="display: flex;
+                  justify-content: center;
+                  width: 90%;
+                  background-color: rgba(0, 0, 0, 0.7);
+                  color: white;
```

Comment:
```suggestion
                  background-color: var(--onyx-color-base-neutral-800);
                  color: var(--onyx-color-text-icons-neutral-intense);
```

Reply:
I used -900 for the background color, see also in the demo app so the contrast was a bit better in combination with the opacity (Jannick wanted them to be opaque).

Let's see how the final OnyxToast will look like ^^ we'll swap it out when it comes.